### ISSUE TITLE
[Bug]Mangletooth's "Hello" window

### ISSUE THREAD

**Issue by Crysicle** [TrinityCore/TrinityCore#5144]
Bug:
Manglethorn says some things to you which he should say only after you complete some of his quests. I'm not sure after which quest the text he says should change, but i think it's after Blood Shards of Agamaggan because after you complete that quest, he offers you other quests which allow you to gain buffs in exchange for the Blood Shards.

Links:
Mangletooth - http://old.wowhead.com/npc=3430
A better look at his quests - http://www.wowwiki.com/Mangletooth_(pre-Cataclysm)
Things he says before completing any quests - http://img803.imageshack.us/img803/4825/wowscrnshot020812210548.jpg

Core revision: Trinity Core 7a117dd42966+
Database revision: TDB 335.11.45
Addons: Anticheat1

**Comment by Killyana** [comment]
``` sql
DELETE FROM `gossip_menu` WHERE `Entry`=2944 AND `text_id`=3656;
INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES (2944, 3656);

DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=14 AND `SourceGroup`=2944;
INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,`ElseGroup`,`ConditionTypeOrReference`,`ConditionValue1`,`ConditionValue2`,`ConditionValue3`, `ErrorTextId`,`ScriptName`,`Comment`) VALUES
(14,2944,3670,0,8,906,0,0,0,'','Show gossip menu text if player finish Betrayal from Within');

**Comment by fvicaria** [comment]
There are a bunch of quests that Mangletooth is supposed to reward you with a buff when you turn the quest in:
[20] Agamaggan's Agility (repeatable)
[20] Agamaggan's Strength (repeatable)
[20] Razorhide (repeatable)
[20] Rising Spirit (repeatable)
[20] Spirit of the Wind (repeatable)
[20] Wisdom of Agamaggan (repeatable)

![untitled](https://cloud.githubusercontent.com/assets/9414661/8023439/9d8882dc-0d04-11e5-96b7-aeebe5264255.png)

![untitled2](https://cloud.githubusercontent.com/assets/9414661/8023446/f0d81bfa-0d04-11e5-9429-fb6b7df8f796.png)

The quests work fine but no buff is given as a reward.

TDB 335.58

Let me know if I need to open a new issue for this with more info.

**Comment by tkrokli** [comment]
@fvicaria : your issue is different from the original issue. The original issue is about conditions for only seeing those repeatable quests to gain the buffs, not about the buffs themselves.

From my point of view, your post belongs in a new issue, unless there are any open ones on this already. Searching for "Mangletooth" or Agamaggan, I can only find closed ones :
https://github.com/TrinityCore/TrinityCore/search?q=Mangletooth&type=Issues&utf8=✓
https://github.com/TrinityCore/TrinityCore/search?q=Agamaggan&type=Issues&utf8=✓

**Comment by Killyana** [comment]
Because the npc is inside a gob, so out of sight, just open the cage and complete the quest and you will get the buff.

**Comment by fvicaria** [comment]
So obvious. Apologies.
